agents/agents/executor/db/crosstable_test.go

```diff
@@ -88,20 +89,21 @@ func (t *DBSuite) TestGetTimestampForMessage() {
 		err = testDB.StoreAttestation(t.GetTestContext(), attestationC, origin+1, 3, 1)
 		Nil(t.T(), err)
 
-		timestamp, err := testDB.GetTimestampForMessage(t.GetTestContext(), origin, origin+1, nonceA, "")
+		retrievedTimestampA, err := testDB.GetTimestampForMessage(t.GetTestContext(), origin, origin+1, nonceA)
 		Nil(t.T(), err)
-		Equal(t.T(), uint64(3), *timestamp)
+		Equal(t.T(), uint64(3), *retrievedTimestampA)
 
-		timestamp, err = testDB.GetTimestampForMessage(t.GetTestContext(), origin, origin+1, nonceB, "")
+		retrievedTimestampB, err := testDB.GetTimestampForMessage(t.GetTestContext(), origin, origin+1, nonceB)
 		Nil(t.T(), err)
-		Equal(t.T(), uint64(3), *timestamp)
+		Equal(t.T(), uint64(3), *retrievedTimestampB)
 
-		timestamp, err = testDB.GetTimestampForMessage(t.GetTestContext(), origin, origin+1, nonceC, "")
+		retrievedTimestampC, err := testDB.GetTimestampForMessage(t.GetTestContext(), origin, origin+1, nonceC)
 		Nil(t.T(), err)
-		Equal(t.T(), uint64(1), *timestamp)
+		Equal(t.T(), uint64(1), *retrievedTimestampC)
 	})
 }
 
+// TODO: Add more edge cases.
```

Comment:
For these tests, the edge cases I had in mind were just altering the origin/destination chains, but now that I think about it, I think it might over-obfuscate the main thing that these tests are trying to test.